### PR TITLE
Task/gv 1050 add tooltips to terms all pages

### PR DESCRIPTION
**Pull request by Anchello**

### CODE REVIEW

File: packages/shared/components/statistic-item/statistic-item.tsx [hunk 1]
```diff
@@ -6,24 +6,10 @@ import NumberFormat from "react-number-format";
 import withLoader from "shared/decorators/with-loader";
 import { formatCurrencyValue } from "shared/utils/formatter";
 
-enum ITEM {
-  LABEL = "LABEL",
-  VALUE = "VALUE"
-}
+import { HORIZONTAL_POPOVER_POS } from "../popover/popover";
```
Comment (pavelzubov) [inline packages/shared/components/statistic-item/statistic-item.tsx]: Change way to "shared/..."

File: packages/shared/components/statistic-item/statistic-item.tsx [hunk 2]
```diff
@@ -6,24 +6,10 @@ import NumberFormat from "react-number-format";
...
+import Tooltip from "../tooltip/tooltip";
```
Comment (pavelzubov) [inline packages/shared/components/statistic-item/statistic-item.tsx]: Change way to "shared/..."

File: packages/shared/components/statistic-item/statistic-item.tsx [hunk 3]
```diff
@@ -64,15 +65,20 @@ const _StatisticItem: React.FC<IFollowStatisticItemProps> = ({
...
-        <div
-          className={
-            "statistics-item__top " + generateClasses(ITEM.LABEL, invert)
-          }
-        >
-          {label}
-        </div>
-      )}
+      {label ? (
+        labelTooltip && label ? (
```
Comment (pavelzubov) [inline packages/shared/components/statistic-item/statistic-item.tsx]: This is bad.
Try to find solution that will look good and be well supported.

File: packages/shared/components/gv-selection/gv-switch/index.tsx [hunk 4]
```diff
@@ -60,13 +48,28 @@ class GVSwitch extends React.Component<GVSwitchProps> {
...
       touched,
+      labelTooltip,
       disabled,
       ...other
     } = this.props;
     return (
       <span className="gv-switch-wrapper">
-        {this.renderLabel()}
+        {label ? (
+          labelTooltip ? (
```
Comment (pavelzubov) [inline packages/shared/components/gv-selection/gv-switch/index.tsx]: This is bad.
Try to find solution that will look good and be well supported.

File: packages/shared/components/funds/fund-details/fund-details-history-section/fund-structure/fund-structure.tsx [hunk 5]
```diff
@@ -46,6 +49,23 @@ class FundStructure extends React.Component<
     this.fetchFundStructure();
   }
 
+  renderHeaderContent = (column: SortingColumn): JSX.Element => {
```
Comment (pavelzubov) [inline packages/shared/components/funds/fund-details/fund-details-history-section/fund-structure/fund-structure.tsx]: Move it to separate FC

File: packages/shared/components/details/details-description-section/details-investment/details-investment.tsx [hunk 6]
```diff
@@ -77,6 +77,8 @@ class DetailsInvestment extends PureComponent<
 
     const profitValue = personalDetails.value - personalDetails.invested;
 
+    console.log(asset);
```
Comment (pavelzubov) [inline packages/shared/components/details/details-description-section/details-investment/details-investment.tsx]: Remove it

File: packages/shared/components/wallet/components/wallet-container/wallet-container.tsx [hunk 7]
```diff
@@ -19,6 +21,15 @@ import TransactionsRow from "../wallet-transactions/transactions-row";
 import WalletTransactions from "../wallet-transactions/wallet-transactions";
 import { WALLET_TRANSACTIONS_COLUMNS } from "../wallet-transactions/wallet-transactions.constants";
 
+const renderTooltipTab = (tooltipContent: string, tabLabel: string) => (
```
Comment (pavelzubov) [inline packages/shared/components/wallet/components/wallet-container/wallet-container.tsx]: Change to FC

**Comment by pavelzubov** [comment]
And question mark is shown on value too.
image

**Comment by Anchello** [comment]
> And question mark is shown on value too.
> image

The tooltip for entry fee in the details program was before this task, it shows when entryFeeSelected !== entryFeeCurrent

File: packages/shared/components/statistic-item/statistic-item.tsx (hunk 3 above)
Comment (Anchello) [reply packages/shared/components/statistic-item/statistic-item.tsx]: Reverted changing in StatisticItem and added new component TooltipLabel which is used with prop of label

File: packages/shared/components/gv-selection/gv-switch/index.tsx (hunk 4 above)
Comment (Anchello) [reply packages/shared/components/gv-selection/gv-switch/index.tsx]: Reverted changing in GVSwitch and added new component TooltipLabel which is used with prop of label

**Comment by pavelzubov** [comment]
> The tooltip for entry fee in the details program was before this task, it shows when entryFeeSelected !== entryFeeCurrent
Yes. But this does not mean that the question mark should be displayed not only above the label, but also above the value)

File: packages/shared/components/funds/fund-details/fund-details-history-section/fund-structure/fund-structure-header-content.tsx [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    )}
+  >
+    {t(`fund-details-page.history.structure.${column.name}`)}
+  </span>
+);
+
+const FundStructureHeaderCell = React.memo(
+  translate()(_FundStructureHeaderCell)
+);
+export default FundStructureHeaderCell;
```
Comment (pavelzubov) [inline packages/shared/components/funds/fund-details/fund-details-history-section/fund-structure/fund-structure-header-content.tsx]: Try to use the same words when naming components and files (rename file to fund-structure-header-cell or component to FundStructureHeaderContent).

File: packages/shared/components/wallet-widget/wallet-widget.tsx [hunk 9]
```diff
@@ -86,7 +86,7 @@ class _WalletWidget extends React.PureComponent<
               </StatisticItem>
             </div>
             <div className="wallet-details__item">
-              <StatisticItem label={t("wallet-widget.invested-value")}>
+              <StatisticItem label={t("wallet-widget.invested")}>
```
Comment (pavelzubov) [inline packages/shared/components/wallet-widget/wallet-widget.tsx]: It breaks the path to translation.
image

File: packages/shared/components/tooltip-label/tooltip-label.tsx [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+  labelText: string;
+  className?: string;
+}> = React.memo(({ tooltipContent, labelText, className }) => (
+  <Tooltip
+    horizontal={HORIZONTAL_POPOVER_POS.LEFT}
+    render={() => <div className="tooltip__content">{tooltipContent}</div>}
+  >
+    <span className={classNames("tooltip__label", className)}>{labelText}</span>
+  </Tooltip>
+));
```
Comment (pavelzubov) [inline packages/shared/components/tooltip-label/tooltip-label.tsx]: This is good.

File: packages/shared/components/funds/fund-details/fund-details-history-section/fund-structure/fund-structure-header-content.tsx (hunk 8 above)
Comment (Anchello) [reply packages/shared/components/funds/fund-details/fund-details-history-section/fund-structure/fund-structure-header-content.tsx]: Done

File: packages/shared/components/tooltip-label/tooltip-label.tsx (hunk 10 above)
Comment (Anchello) [reply packages/shared/components/tooltip-label/tooltip-label.tsx]: Done

**Pull request merged**